cli/src/config.rs

```diff
@@ -589,8 +589,12 @@ pub fn default_config_layers() -> Vec<ConfigLayer> {
         parse(include_str!("config/revsets.toml")),
         parse(include_str!("config/templates.toml")),
     ];
-    if cfg!(unix) {
-        layers.push(parse(include_str!("config/unix.toml")));
+    if cfg!(target_os = "macos") {
+        layers.push(parse(include_str!("config/macos.toml")));
+    } else if cfg!(target_os = "linux") {
+        layers.push(parse(include_str!("config/linux.toml")));
+    } else if cfg!(unix) {
+        layers.push(parse(include_str!("config/other_unix.toml")));
```

Comment:
I agree with Ilya; lets just get rid of all this and use `nano` everywhere. There is very little reason someone would complain here unless they used pico(?), in which case they could just set it back. `nano` is objectively a better choice everwhere except Windows, IMO.

Reply:
> `nano` is objectively a better choice everwhere except Windows, IMO.

It's also the only place where it doesn't exist by default.